rollup.config.js

```diff
@@ -869,7 +869,8 @@ export default function rollup(options) {
 }
 
 async function triggerLiveReload(appDir) {
-  // tickle live reload by touching the server entry
+  // Tickle live reload by touching the server entry.  Include all extensions
+  // since they may or may not use TS and/or JSX
```

Comment:
nit: I think explicitly calling out that all these extensions are valid is good, but explaining _why_ they are valid is the tricky part.

Maybe something like:

```js
// Tickle live reload by touching the server entry.
// Consider all of `entry.server.{tsx,ts,jsx,js}` since React may be used via `React.createElement` without the need for JSX.
```